discordkit/utils/command.ts

```
@@ -0,0 +1,267 @@
+                  (acc, option) => {
+                    if ("value" in option) {
+                      return { ...acc, [option.name]: option.value };
+                    }
+                    return acc;
+                  },
+                  {} as Record<string, unknown>,
+                ) as T;
+              }
+
+              if (config.slashCommand) {
+                await this.execute(
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.

(lint/complexity/noThisInStatic)